README.md

```diff
@@ -330,6 +330,21 @@ accessories:
 
 Now run `mrsk accessory start mysql` to start the MySQL server on 1.1.1.3. See `mrsk accessory` for all the commands possible.
 
+### Using Cron
+
+You can use a specific container to run your Cron jobs:
+
+```yaml
+servers:
+  cron:
+    hosts:
+      - 192.168.0.1
+    cmd:
+      bash -c "cat config/crontab | crontab - && cron -f"
```

Comment:
are you running you cron container as root? Because there seems to be some massage to make this command work as non-root user, I couldn't figure out..

Reply:
I am also struggling to make logs of cron jobs available to `docker logs`, I tried a couple of things (like [this one](https://betterstack.com/community/questions/how-to-run-cron-jobs-inside-docker-container/) for example) but none seems to work

Reply:
I ended up running as root, you can check my write-up on this [blog post](https://glaucocustodio.github.io/2024/01/23/migrating-from-dokku-to-kamal-scheduling-cron-jobs)